build-system/tasks/extension-generator/extension-doc.template.md

```diff
@@ -1,19 +1,14 @@
 <!--
-  1. Change "category" below to one of:
-       ads-analytics
-       dynamic-content
-       layout
-       media
-       presentation
-       social
-  2. Remove any of the "formats" that don't apply.
-     You can also add the "ads" and "stories" formats if they apply.
-  3. And remove this comment! (no empty lines before "---")
+  All documentation starts with frontmatter. Front matter organizes documentation on amp.dev
+  and improves SEO.
+  * Include the relevant category(ies): ads-analytics, dynamic-content, layout, media, presentation, social, personalization
+  * List applicable format(s): websites, ads, stories, email
+  * Remove this comment! (no empty lines before "---")
 -->
 
 ---
 
-\$category: presentation
+category: presentation
```

Comment:
Should frontmatter start with `category` (like it is now) or `$category` (like it was earlier)?

If it's just `category`, line 79 can be deleted in the code below. (It was needed because the `$` sign had to be escaped to satisfy prettier.)


https://github.com/ampproject/amphtml/blob/185493b6f29f68328f3c194f4bf341a44691cac7/build-system/tasks/extension-generator/index.js#L76-L81

Reply:
Thanks for flagging, I've added the `$` back in, but `gulp prettify --files "extensions/amp-crystal/**.md"` flags a few errors, but they seem inconsistent. Any insight? 

<img width="1166" alt="Screen Shot 2019-11-14 at 4 18 50 PM" src="https://user-images.githubusercontent.com/23297417/68906864-959e6d80-06fa-11ea-942f-83209606632c.png">

Reply:
This means the `md` file that's created by copying the template is no longer pretty.

Perhaps you can run `gulp prettier --files build-system/tasks/extension-generator/extension-doc.template.md`  and see if it's able to fix the template?

Reply:
Fixed it - there was a spacing issue, and I forgot to delete a comment manually. Thanks for your help!